# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1757628159 -7200
# Parent  3994549a1f86cc9c53baec9a486caa562699b708
Misc: Switch off new Account Hub (too buggy) and fix Windows font rendering (bug 1946024).

A really bad choice was made for calendar.alarms.showmissed in bug 1934069 leading to missed reminders, bug 1988288.

diff --git a/calendar/base/calendar.js b/calendar/base/calendar.js
--- a/calendar/base/calendar.js
+++ b/calendar/base/calendar.js
@@ -32,9 +32,10 @@ pref("calendar.events.defaultActionEdit"
 pref("calendar.agenda.days", 14);
 
 // alarm settings
 pref("calendar.alarms.show", true);
-pref("calendar.alarms.showmissed", false);
+// Must not be set to false, or you miss reminders for future events.
+pref("calendar.alarms.showmissed", true);
 pref("calendar.alarms.playsound", true);
 pref("calendar.alarms.soundType", 0);
 pref("calendar.alarms.soundURL", "chrome://calendar/content/sound.wav");
 pref("calendar.alarms.defaultsnoozelength", 5);
diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -1552,9 +1552,10 @@ pref("mail.thread.conversation.enabled",
 pref("mail.dark-reader.enabled", true);
 pref("mail.dark-reader.show-toggle", true);
 
 // Enable the new account setup (starting from the second account)
-pref("mail.accounthub.enabled", true);
+// This has too many errors.
+pref("mail.accounthub.enabled", false);
 
 // Enable address book setup via account hub
 pref("mail.accounthub.addressbook.enabled", false);
 
@@ -1571,4 +1572,9 @@ pref("calendar.dialogs.new.enabled", fal
 pref("mail.threadpane.listview", 1);
 
 // Row count for the cards view, currently bound to a range between 2 and 3.
 pref("mail.threadpane.cardsview.rowcount", 3);
+
+// Fit font rendering.
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_max_size", 15);
+pref("gfx.font_rendering.cleartype_params.force_gdi_classic_for_families",
+  "Arial,Consolas,Courier New,Microsoft Sans Serif,Segoe UI,Tahoma,Trebuchet MS,Verdana");
